api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirEncounterServiceImpl.java

```diff
@@ -37,8 +39,8 @@
 	
 	@Override
 	@Transactional(readOnly = true)
-	public Encounter getEncounterByUuid(String uuid) {
-		return translator.toFhirResource(dao.getEncounterByUuid(uuid));
+	public Encounter get(String uuid) {
+		return translator.toFhirResource(dao.get(uuid));
 	}
```

Comment:
This shouldn't be necessary to have here at all.

Reply:
thanks ,,have removed it